src/mscorlib/model.xml

```diff
@@ -151,6 +151,9 @@
       <Member Name="SetSwitch(System.String,System.Boolean)" />
       <Member Name="TryGetSwitch(System.String,System.Boolean@)" />
       <Member Name="GetData(System.String)" />
+      <Member MemberType="Event" Name="UnhandledException" />
+      <Member Name="add_UnhandledException(System.UnhandledExceptionEventHandler)" />
```

Comment:
It should not be necessary to list the add/remove handlers explicitly. None of the other events in model.xml have them listed.